op-node/node/api.go

```diff
@@ -106,6 +107,10 @@ func (n *adminAPI) ConductorEnabled(ctx context.Context) (bool, error) {
 	return n.dr.ConductorEnabled(ctx)
 }
 
+func (n *adminAPI) SetRecoverMode(ctx context.Context, mode bool) error {
+	return n.dr.SetRecoverMode(ctx, mode)
```

Comment:
When this is set, does it actually propagate through to the l1 origin selector? I'm not sure, it may only be set in the sequencer struct. But this is only read during setup, right?